morpheushelper/cogs/discord_bot_tokens.py

```diff
@@ -0,0 +1,45 @@
+from PyDrocsid.translations import translations
+from discord import Embed, Member, Message, Forbidden, NotFound, HTTPException
+from discord.ext import commands
+from discord.ext.commands import Cog, Bot, Context, guild_only
+
+from colours import Colours
+import re
+
+
+def make_embed(requested_by: Member) -> Embed:
+    embed = Embed(title=translations.discordbottoken_title, colour=Colours.DiscordBotToken)
+    embed.description = translations.discordbottoken_description
+    embed.set_footer(text=translations.f_requested_by(requested_by, requested_by.id), icon_url=requested_by.avatar_url)
+    return embed
+
+
+class DiscordBotTokenCog(Cog):
```

Comment:
Please consider to create a class var with a compiled version of the regex, like so:
```python
class DiscordBotTokenCog(Cog):
    RE_DC_TOKEN = re.compile(r'[A-Za-z\d]{24}\.[A-Za-z\d]{6}\.[A-Za-z\d\-\_]{27}')
```